### ISSUE TITLE
Retain back button in company details on manual page reload

### ISSUE THREAD

**Issue by emil-ronkainen** [careerfairsystems/nexpo-app#283]
**Issue:**
When manually reloading the app (f5) on a company details page, the back button disappears.

**Repro Steps:**
1. Log in to the application
2. Navigate to a company under the *companies* tab.
3. Reload the page (e.g. using f5 or similar)

**Expected behavior:**
The button that navigates back to *companies* is still on the page
<img width="255" alt="image" src="https://user-images.githubusercontent.com/47136996/233326186-6edbd89c-6a20-4d33-a3e8-96825685346c.png">

**Actual behavior:**
The button is no longer present on the page
<img width="315" alt="image" src="https://user-images.githubusercontent.com/47136996/233326041-3d5b1af5-1cfe-40b4-bce0-edd6c1ea1373.png">

It might be noteworthy that the *event* and *student session* tabs circumvent this bug by navigating to the *companies* tab on manual reload.

**Comment by Stronkness** [comment]
Should the event and student session navigate to companies tab during reload is the question also

**Comment by emil-ronkainen** [comment]
That's a great point. A possible solution to this could be making sure the companies tab behave as the others, setting that as the closing criteria for this issue, and adding a new issue for revising the overall back button behavior. Preferrably, we set the new issue as blocked by this one to make sure all back buttons behave similarly before (possibly) changing the way they work.

**Comment by Stronkness** [comment]
It has been found that reload is only applicable on web-testing, therefore close issue
